cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -192,6 +194,21 @@ def gen_arg_parser(cls, args_to_parse=None):
             args_to_parse = sys.argv[1:]
         return argP.parse_args(args_to_parse)
 
+    def validate_rgw_endpoint_tls_cert(self):
+        cmd = ['openssl', 'x509', '-in', self._arg_parser.rgw_tls_cert_path]
```

Comment:
This won't work if the expectation is to get a `.pem` with the server key, server certificate, and any other CA or intermediate certificates.